test_unstructured/partition/pptx/test_pptx.py

```diff
@@ -89,12 +89,6 @@ def test_partition_pptx_from_file_with_metadata_filename():
         assert element.metadata.filename == "test"
 
 
-def test_partition_pptx_raises_with_both_specified():
```

Comment:
This behavior is no longer implemented. If both filename and file are provided, filename is used.